test/test_integration_credentials_netrc.py

```diff
@@ -373,16 +345,16 @@ def test_cli_configured_in_insecure_mode_but_run_in_secure_mode_raises_error(sel
         self.assertIn("The client was initialized without", output)
 
     @integration_test()
-    def test_simple_login(self, keystore_disable_mock):
+    def test_simple_login_netrc(self, keystore_disable_mock):
```

Comment:
I did a count of how many `def`s we have between the new netrc and the old netrc integration test file. The old one has 40 and the new one has 39. Do you know why?